tests/infrastructure/test_ialirt_ingest_lambda_construct.py

```diff
@@ -9,26 +9,55 @@ def populate_table(table):
     """Populate DynamoDB table."""
     items = [
         {
+            "apid": 478,
             "met": 123,
-            "ingest_time": "2021-01-01T00:00:00Z",
+            "ingest_date": "2021-01-01T00:00:00Z",
             "packet_blob": b"binary_data_string",
         },
         {
+            "apid": 478,
             "met": 124,
-            "ingest_time": "2021-01-01T00:00:01Z",
+            "ingest_date": "2021-02-01T00:00:00Z",
             "packet_blob": b"binary_data_string",
         },
     ]
     for item in items:
         table.put_item(Item=item)
 
-    return item
+    return items
 
 
-def test_query_by_sct_vtcw(table, populate_table):
-    """Test to query irregular packet length."""
-    response = table.query(KeyConditionExpression=Key("met").eq(124))
+def test_query_by_met(table, populate_table):
+    """Test to query by met."""
+    expected_items = populate_table
 
+    response = table.query(KeyConditionExpression=Key("apid").eq(478))
+
+    items = response["Items"]
+
+    for item in range(len(items)):
+        assert items[item]["apid"] == expected_items[item]["apid"]
+        assert items[item]["met"] == expected_items[item]["met"]
+        assert items[item]["ingest_date"] == expected_items[item]["ingest_date"]
+        assert items[item]["packet_blob"] == expected_items[item]["packet_blob"]
```

Comment:
Can you compare the entire `items == expected_items` or each individual item? This will help if you add/remove attributes in the future and not needing to update everything here too.
```suggestion
        assert items[item] == expected_items[item]
```